### ISSUE TITLE
when running spec when `integrationFolder` default is overridden, Cypress looks in wrong directory path.

### ISSUE THREAD

**Issue by jennifer-mann** [cypress-io/cypress#117]
When I've set `integrationFolder` in `cypress.json` to be the following:

```javascript
{
  "integrationFolder": "foo"
}
```

And my folder structure is this:
```text
+ projectRoot
  + foo
     fake.js
```

When I run `fake.js` within the Cypress webapp, there is an error displayed in the runner: 

```
Sorry, there's something wrong with this file:

.... / projectRoot / foo / integration / fake.js

```

Cypress is trying to run this test in a nested `integration` folder that I didn't specify in the config `integrationFolder` path.

**Comment by brian-mann** [comment]
This isn't a bug. The problem is the file. 

The only thing I see that needs updating is that the path name has 'integration' as a segment, which is wrong. That's just cosmetic but will be fixed in `0.15.1`

**Comment by jennifer-mann** [comment]
The UI error now properly prints the integration test file's path.

Fixed in [`0.15.1`](https://github.com/cypress-io/cypress/wiki/Changelog#0151-04012016).
